user_management/api/views.py

```diff
@@ -154,11 +152,11 @@ def initial(self, request, *args, **kwargs):
         try:
             self.user = User.objects.get(pk=uid)
         except User.DoesNotExist:
-            raise AuthenticationFailed(detail=self.message)
+            raise exceptions.InvalidExpiredToken(detail=self.message)
 
         token = kwargs['token']
         if not default_token_generator.check_token(self.user, token):
-            raise AuthenticationFailed(detail=self.message)
+            raise exceptions.InvalidExpiredToken(detail=self.message)
```

Comment:
The `InvalidExpiredToken` exception has the same default message as you're adding here.